src/main/webapp/script.js

```diff
@@ -38,3 +38,31 @@ function createCommentElement(comment) {
 function addParagraph(content) {
   return "<p>" + content + "</p>";
 }
+
+function newGroupchat() {
+  const request = new Request("/load-groupchat", {method: 'POST'});
```

Comment:
I had no idea you could make a request directly in Javascript like this. I learn something new every day!